test/forward/thrift.js

```diff
@@ -29,9 +29,14 @@ var someSpec = fs.readFileSync(
 var allocCluster = require('../lib/test-cluster.js');
 
 allocCluster.test('register and forward with thrift', {
-    size: 5
+    size: 5,
+    namedRemotes: ['echoRemote'],
+    remoteSpecs: {
```

Comment:
Nope; remotes. I want to add other thigns to it like amount and other things so we can deprecate the jank array.